content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -15,6 +15,48 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <!-- more -->
 
+## Deferred Rendering
+
+<div class="release-feature-authors">authors: @DGriffin91</div>
+
+The two most popular "rendering styles" are:
+
+* **Forward Rendering**: do all material/lighting calculations in a single render pass
+  * **Pros**: Simpler to work with. Works on / performs better on more hardware. Supports MSAA. Handles transparency nicely.
+  * **Cons**: Lighting is more expensive / fewer lights supported in a scene, some rendering effects are impossible (or harder) without a prepass
+* **Deferred Rendering**: do one or more pre-passes that collect relevant information about a scene, then do material/lighting calculations in _screen space_ in a final pass after that.
+  * **Pros**: Enables some rendering effects that are not possible in forward rendering. This is especially important for GI techniques. Can support more lights in a scene
```

Comment:
Regarding the number of lights, this can be true, though we use the same clustered impl for deferred (and clustered deferred rendering is somewhat typical). I would expect that lots of lights are probably still faster with deferred since it's all just done in the one pass and there's no overdraw. I don't have an issue with what's said here, but I guess it's just more that the performance with lots of lights would be better rather than a limitation on how many can be used. (Sorry I'm mostly just thinking out loud here)

Reply:
> but I guess it's just more that the performance with lots of lights would be better rather than a limitation on how many can be used. (Sorry I'm mostly just thinking out loud here) 

Yeah I see these as two sides of the same coin. Ex: clustered doesn't inherently increase the number of lights _supported_ (provided you are using storage buffers, which is a separate concern), it just reduces the number of lights being processed per fragment, increasing the number of lights _you can reasonably use_. Same story for deferred lighting: it reduces the amount of work done by only lighting visible fragments, increasing the number of lights you can reasonably use.

I think "support more lights in a scene" is a reasonably terse way to express that to a layperson, but I'm open to alternatives!

Reply:
Makes sense to me, I like the current terseness  :)